cmake/option/option_gbs-tizen.cmake

```diff
@@ -0,0 +1,32 @@
+# Copyright 2018 Samsung Electronics Co., Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# linux common
+include("cmake/option/option_unix_common.cmake")
+include("cmake/option/option_linux_common.cmake")
+
+# gbs-tizen specific
+if(DEFINED TARGET_BOARD)
+  if(${TARGET_BOARD} STREQUAL "rpi3")
+    set(FLAGS_COMMON
+          ${FLAGS_COMMON}
+          "-mfloat-abi=softfp"
+          "-mfpu=neon-vfpv4"
+          )
+  else()
+    message(FATAL_ERROR "TARGET_BOARD=`${TARGET_BOARD}` is unknown to make")
+  endif()
+else()
+  message(FATAL_ERROR "TARGET_BOARD is undefined")
+endif()
```

Comment:
@haesik What about `option_arm-tizen-gbs.cmake`?

Reply:
I don't agree with your opinion.. Currently, considering gbs as an architecture is now easy to support gbs build.. if we uas config_arm-tizen-gbs, we should make config_arm-tizen-gbs, config_x86-tizen-gbs, config_x86_64-tizen-gbs, etc,,.. for the same gbs build.

Reply:
@haesik I also know considering `gbs` as an architecture may be easiest way to get it working. But `gbs` is not an architecture. And it is open source that many developers work on same code base. I don't want to break the rule for reducing work on tizen. Alternatively, we may use `option_noarch-tizen-gbs.cmake` if a common file can be used for multiple targets.

Reply:
I think you don't want to consider gbs as architecture.. I agree to use noarch for gbs.
the file name could be option_noarch-tizen.cmake